lib/rubocop/rspec/language.rb

```diff
@@ -159,9 +159,14 @@ def context(element)
       end
 
       module Runners # :nodoc:
-        ALL = %i[to to_not not_to].freeze
-        def self.all(element)
-          ALL.include?(element)
+        class << self
+          def to_a
+            %i[to to_not not_to].freeze
+          end
```

Comment:
```suggestion
        ALL = %i[to to_not not_to].freeze
        class << self
          def to_a
            ALL
          end
```

This was a constant initially to save a penny on creating a 3-element array each time we access it.